modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase

```diff
@@ -2582,7 +2582,7 @@ definition {
 
 		ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject287");
 
-		for (var number : list "1,2,3,4,5,6,7,8,9,10,011,012,013,014") {
+		for (var number : list "1,2,3,4,5,6,7,8") {
```

Comment:
The paging behavior changed, so it was necessary to change the tests.
I talked about the behavior with Gabriel Albuquerque via Slack.